tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php

```diff
@@ -424,6 +499,265 @@ public function provideSemicolonForChainedCallsFixCases()
                         ->method2();
                 ?>',
             ],
+            [
+                '<?php
+
+                    self
+                        ::method1()
+                        ->method2()
+                    ;
+                ?>',
+                '<?php
+
+                    self
+                        ::method1()
+                        ->method2();
+                ?>',
+            ], [
+                '<?php
+
+                    self
+                        ::method1()
+                        ->method2() // comment
+                    ;
+
+
+',
+                '<?php
+
+                    self
+                        ::method1()
+                        ->method2(); // comment
+
+
+',
+            ], [
+                '<?php
+
+                    Service::method1()
+                        ->method2()
+                    ;
+
+                    Service::method3();
+                    $this
+                        ->method1()
+                        ->method2()
+                    ;',
+                '<?php
+
+                    Service::method1()
+                        ->method2()
+                    ;
+
+                    Service::method3();
+                    $this
+                        ->method1()
+                        ->method2();',
+            ], [
+                '<?php
+
+                    Service
+                        ::method2()
+                    ;
+                ?>',
+                '<?php
+
+                    Service
+                        ::method2();
+                ?>',
+            ], [
+                '<?php
+
+                    Service::method1()
```

Comment:
Should it support also multiple static calls?
```php
Service::method1()
    ::method2()
    ->method3()
;
```

Reply:
What would `method1` return to allow that?

Reply:
Ah like something that returns a class string, I think it should be allowed but I want to know what others think

Reply:
Yes, something like this.. not sure if there are any real-life cases for it.
```php
<?php

class A 
{
    public static function fa()
    {
        return 'B';
    }
}

class B
{
    public static function fb()
    {
        return new C();
    }
}

class C
{
    public function fc()
    {
        return 'c';
    }
}

echo A::fa()
    ::fb()
    ->fc();

```

Reply:
Added some tests for chained static calls, thanks @egircys